zerver/decorator.py

```diff
@@ -266,7 +266,7 @@ def _wrapped_func_arguments(request, api_key=REQ(),
                 if request.content_type == 'application/json':
                     request_body = ujson.dumps(ujson.loads(request.body), indent=4)
                 else:
-                    request_body = str(request.body)
+                    request_body = request.body.decode()
```

Comment:
In Python3, `str(b'a') == "b'a'"`, if you want `'a'`, then you must call `decode()`. In Python 2, decode makes it unicode.